src/reports/library/io_writer.hpp

```diff
@@ -0,0 +1,29 @@
+#pragma once
+#include <vector>
+#include <memory>
+
+enum KindWriter {HDF5, ASCII, MPIIO};
```

Comment:
This could be `enum class IoWriter::Kind`

Reply:
I didnt know about enum class, thanks!